ex03.ex

```diff
@@ -149,7 +163,25 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+  def won({:x,_a,_b,:x,_c,_d,:x,_e,_f}), do: :x
```

Comment:
## -2

You could reduce the number of comparisons in half by writing

~~~ elixir
def won(p, p, p, _a, _b, _c, _d, _e, _f}), do: p
~~~